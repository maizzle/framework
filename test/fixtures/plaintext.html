<plaintext>Will be removed</plaintext>
<div>Will be preserved</div>
