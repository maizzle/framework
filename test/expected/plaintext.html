
<div>Will be preserved</div>
